german/finger-ins-loch/ins-Wasser/davonschwimmen/davonschwimmen.md

```diff
@@ -0,0 +1 @@
+Sie drehen sich in die dem Hai entgegengesetzte Richtung und schwimmen so schnell Sie können in diese Richtung. Plötzlich sehen Sie am Horizont eine Insel.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.

Reply:
Split the line into two.